sw/device/silicon_creator/rom/keys/real/rsa/hsmtool/earlgrey_keygen.hjson

```diff
@@ -0,0 +1,259 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+// The earlgrey keys were generated by the following command:
+// hsmtool -t <token> -u user -p <pin> exec earlgrey_keygen.hjson
+
+[
+  ////////////////////////////////////////////////////////////
+  //
+  // There are 3 prod keys named earlgrey_a0_prod_{0,1,2}
+  //
+  ////////////////////////////////////////////////////////////
+  {
```

Comment:
This can be done in a followup but we should define the templates once and refer to them from the keys to have a single source of truth.

Reply:
I agree, however this hjson file is what was used to do key creation and I feel a bit leery about changing it after the fact.

We _should_ be able to place the templates into individual files and then reference them using `@filename` like notation. 

Created #18846 to track and follow-up.